### DISCUSSION TITLE
Not getting updates in new installer

### DISCUSSION THREAD

**Discussion by studio051** [flybywiresim/aircraft#4774] [Help] [answered]
I’m seeing many cashes being made at Github to the aircraft code but my installer has t updated anything since 2 days . Can anyone comment ?

**Comment by tracernz** [comment]
Are you looking at the development version? That is the only version receiving constant updates.

**Comment by JARVG** [comment]
And how about the experimental version, is that updated lately?

**Reply by tracernz** [reply]
> And how about the experimental version, is that updated lately?

No, as announced the autopilot/FBW work that was ongoing in experimental has been merged into master(development), so experimental is not used at the moment.

**Reply by JARVG** [reply]
Ok thnx

**Comment by FGuez** [comment]
Hi, 
Looks like my Sim isn't adding the new update 0.6.1 any helps or inputs please .

**Reply by scoopscode** [reply]
Are you choosing the correct aircraft when loading in?

**Comment by studio051** [comment, marked as answer]
Issue resolved . Thank you
